ClearKeep/Scenes/ChatGroup/ChatGroupView.swift

```diff
@@ -0,0 +1,207 @@
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.colorScheme) private var colorScheme
+	@Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
+	@State private(set) var samples: Loadable<[IGroupChatModel]>
+	@State private(set) var searchText: String
+	@State private(set) var searchStyle: TextInputStyle = .default
+	@State private(set) var isShowingView: Bool = false
+	@State private(set) var isSelectedUser: Bool = false
+
+	// MARK: - Init
+	public init(samples: Loadable<[IGroupChatModel]> = .notRequested,
+				searchText: String = "",
+				inputStyle: TextInputStyle = .default) {
+		self._samples = .init(initialValue: samples)
+		self._searchText = .init(initialValue: searchText)
+	}
+	
+	// MARK: - Body
+	var body: some View {
+		VStack(spacing: Constants.paddingVertical) {
+			Button(action: customBack) {
+				HStack(spacing: Constants.spacer) {
+					AppTheme.shared.imageSet.chevleftIcon
+						.aspectRatio(contentMode: .fit)
+						.foregroundColor(foregroundButtonBack)
+					Text("GroupChat.Back.Button".localized)
+						.padding(.all)
+						.font(AppTheme.shared.fontSet.font(style: .body1))
+						.foregroundColor(foregroundButtonBack)
+				}
+				.frame(maxWidth: .infinity, alignment: .leading)
+			}
+			.padding(.top, Constants.spacerTopView)
+
+			SearchTextField(searchText: $searchText,
+							inputStyle: $searchStyle,
+							inputIcon: AppTheme.shared.imageSet.searchIcon,
+							placeHolder: "GroupChat.Search.Placeholder".localized,
+							onEditingChanged: { isEditing in
+				if isEditing {
+					searchStyle = .normal
+				} else {
+					searchStyle = .highlighted
+				}
+			})
+				.padding(.top, Constants.paddingVertical)
+
+			HStack {
+				Text("Alissa Baker".localized)
+					.font(AppTheme.shared.fontSet.font(style: .body3))
+					.foregroundColor(foregroundTagUser)
+					.padding([.leading, .top, .bottom], Constants.paddingTagUser)
+				Button(action: {
+
+				}, label: {
+					Button {
+
+					} label: {
+						AppTheme.shared.imageSet.crossIcon
+							.foregroundColor(foregroundCrossIcon)
+							.padding(.trailing, Constants.paddingTagUser)
+					}
+				})
+			}
+			.background(backgroundTagUser)
+			.cornerRadius(Constants.cornerRadiusTagUser)
+			.frame(maxWidth: .infinity, alignment: .leading)
+			.frame(height: Constants.heightTagUser)
+
+			CheckBoxButtons(text: "GroupChat.User.Add.Title".localized, isChecked: $isShowingView)
+				.frame(maxWidth: .infinity, alignment: .leading)
+
+			if isShowingView {
+				CommonTextField(text: $searchText,
+								inputStyle: $searchStyle,
+								placeHolder: "GroupChat.User.Add.Placeholder".localized,
+								keyboardType: .default,
+								onEditingChanged: { isEditing in
+					if isEditing {
+						searchStyle = .default
+					} else {
+						searchStyle = .highlighted
+					}
+				})
+					.frame(maxHeight: .infinity, alignment: .top)
+				Spacer()
+			} else {
+				VStack {
+					HStack {
+						ZStack {
```

Comment:
Make a list view for list user, Not only using VStack and HStack. Make a customize view for this list.